account_reset_chart/models/res_company.py

```diff
@@ -163,4 +152,30 @@ def unlink_from_company(model):
         unlink_from_company('account.tax.code')
         unlink_from_company('account.journal')
         unlink_from_company('account.account')
+
+        try:
+            self.env['account.partial.reconcile'].search(
+                [('company_id', '=', self.id)]
+            ).unlink()
+        except KeyError:
+            pass
+
+        try:
+            self.env['account.full.reconcile'].search([]).unlink()
```

Comment:
Looks like it could remove more than you want to. But do you need this? https://github.com/odoo/odoo/blob/9.0/addons/account_full_reconcile/models/account_full_reconcile.py#L95